src/GraphQL.php

```diff
@@ -118,6 +119,17 @@ public function schema($schema = null): Schema
      */
     public function query($query, ?array $variables = [], array $opts = []): array
     {
+        $middlewareResponse = app()->make(Pipeline::class)
+            ->send([$query, $variables])
```

Comment:
I think we should add `$opts` here too.

Or, well at least the most important bit would be the `schemaName`.

TBH I despise the whole signature of this `query` method, it just does not make any sense to me.

I think the one from [\Rebing\GraphQL\GraphQLController::executeQuery](https://github.com/rebing/graphql-laravel/blob/26302c195b5dbd29d5d79e72b61cc783da047a39/src/GraphQLController.php#L55) makes much more sense

with the addition of being able to carry over the _parsed query_ (either freshly parsed or retrieved e.g. from APQ).